packages/icon/src/Icon.js

```diff
@@ -94,9 +93,7 @@ export default function Icon(props) {
   );
 }
 
-Icon.AVAILABLE_NAMES = AVAILABLE_NAMES;
-Icon.names = names;
-Icon.sizes = sizes;
+export { AVAILABLE_NAMES, names, sizes };
```

Comment:
This module should only export the `Icon` component specially. Everything else should be exported in `index.js`